src/EventFilter/index.test.js

```diff
@@ -1,54 +1,27 @@
 import React from 'react';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
This test file was broken due to changes in the new versions of redux and react-bootstrap. I needed to redo most of the tests but I made sure to keep the coverage.